lib/image-diff.js

```diff
@@ -54,12 +54,13 @@ ImageDiff.createDiff = function (options, cb) {
   // http://www.imagemagick.org/script/compare.php
   var diffCmd = 'compare';
   var diffArgs = [
-    'compare',
+//    'compare',
```

Comment:
Ah, it looks like that was a duplicate altogether. Can we remove the line instead of commenting it out?